src/adagrams.js

```diff
@@ -1,8 +1,124 @@
-const Adagrams = {
+class Adagrams {
+  //WAVE 1
   drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+    const allLetters = {
+      A: 9,
+      B: 2,
+      C: 2,
+      D: 4,
+      E: 12,
+      F: 2,
+      G: 3,
+      H: 2,
+      I: 9,
+      J: 1,
+      K: 1,
+      L: 4,
+      M: 2,
+      N: 6,
+      O: 8,
+      P: 2,
+      Q: 1,
+      R: 6,
+      S: 4,
+      T: 6,
+      U: 4,
+      V: 2,
+      W: 2,
+      X: 1,
+      Y: 2,
+      Z: 1
+    };
+    let lettersPool = [];
```

Comment:
I **strongly** suggest that you make variables `const` unless you need to reassign the variable.  You can push into a `const` array.  This also applies to a lot of your variables throughout the code here. 

```suggestion
    const lettersPool = [];
```